docs/source/conf.py

```diff
@@ -247,3 +248,47 @@ def setup(app):
     "hankel": ("https://hankel.readthedocs.io/en/latest/", None),
     "emcee": ("https://emcee.readthedocs.io/en/latest/", None),
 }
+
+
+
+# -- Sphinx Gallery Options
+from sphinx_gallery.sorting import FileNameSortKey
+
+sphinx_gallery_conf = {
+    # path to your examples scripts
+    "examples_dirs": [
+        "../../examples/00_intro/",
+        "../../examples/01_random_field/",
+        "../../examples/02_cov_model/",
+        "../../examples/03_variogram/",
+        "../../examples/04_vector_field/",
+        "../../examples/05_kriging/",
+        "../../examples/06_conditioned_fields/",
+        "../../examples/07_transformations/",
+    ],
+    # path where to save gallery generated examples
+    "gallery_dirs": [
+        "examples/00_intro/",
+        "examples/01_random_field/",
+        "examples/02_cov_model/",
+        "examples/03_variogram/",
+        "examples/04_vector_field/",
+        "examples/05_kriging/",
+        "examples/06_conditioned_fields/",
+        "examples/07_transformations/",
+    ],
+    # Patter to search for example files
+    "filename_pattern": r"\.py",
+    # Remove the "Download all examples" button from the top level gallery
+    "download_all_examples": False,
+    # Sort gallery example by file name instead of number of lines (default)
+    "within_subsection_order": FileNameSortKey,
+    # directory where function granular galleries are stored
+    "backreferences_dir": None,
+    # Modules for which function level galleries are created.  In
+    "doc_module": "gstools",
+    #"image_scrapers": ('pyvista', 'matplotlib'),
```

Comment:
Why these comments?

Reply:
For *when* you all start using PyVista for some 3D viz in the example gallery, you will need to add the pyvista image scraper. I figured I would leave this here for when that happens